finalProject/caesarCipher.js

```diff
@@ -0,0 +1,15 @@
+
+// an attempt at solving this, with very little progress
+
+var alphabet = ["A", "B", "C", "D", "E", "F", "G", "H", "I",
+                "J", "K", "L", "M", "N", "O", "P", "Q", "R",
+                "S", "T", "U", "V", "W", "X", "Y", "Z"];
+
+function caesarCipher(string, shiftValue) {
+  var upperCase = string.toUpperCase();
+  var split = upperCase.split("");
+  for (var i = 0; i < alphabet.length; i++) {
```

Comment:
Do you want to loop for the whole alphabet? Or just for each element in your split array? Keep trying to make progress here! Your alphabet array will be very useful for figuring out how to make each character a different character.